README.md

```diff
@@ -6,10 +6,10 @@
 ## Public API
 The services are accessed via an http GET request. All navigation output is generated as GeoJSON ("application/vnd.geo+json")
 
-#### The root is {host}/nldi.
+#### The root is {host}/nldi/api.
 Both a test and a production are exposed to the public:
-__https://cida-test.er.usgs.gov/nldi__ is the test root.
-__https://cida.usgs.gov/nldi__ is the production root.
+__https://labs.waterdata.usgs.gov/nldi/api__ is the test root.
```

Comment:
This looks a little strange since both the test and public urls are the same.

Reply:
Unclear what the new urls are - not anything cida...